assignments/lambda-classes.js

```diff
@@ -1 +1,121 @@
 // CODE here for your Lambda Classes
+// #### Person
+
+// * First we need a Person class. This will be our `base-class`
+// * Person receives `name` `age` `location` `gender` all as props
+// * Person receives `speak` as a method.
+// * This method logs out a phrase `Hello my name is Fred, I am from Bedrock` where `name` and `location` are the object's own props
+
+
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+    }
+    speak() {
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}`);
+    }
+    list() {
```

Comment:
the Person class does not have the favSubjects property. This method has to be inside the Student class.